apps/cowswap-frontend/src/common/pure/Modal/index.cosmos.tsx

```diff
@@ -0,0 +1,98 @@
+import ICON_ARROW from 'assets/icon/arrow.svg'
+import SVG from 'react-inlinesvg'
+import styled from 'styled-components/macro'
+
+import { MOCK_TOKEN, IMAGE_ACCOUNT } from 'common/constants/cosmos';
+import { IconSpinner } from 'common/pure/IconSpinner'
+import { Stepper } from 'common/pure/Stepper'
+
+import { Modal, CowModal, NewModal, NewModalContentTop, NewModalContentBottom } from './index'
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 100vh;
+`
+
+const ArrowRight = styled(SVG)`
+  --size: 12px;
+  width: var(--size);
+  height: var(--size);
+  margin: auto;
+
+  > path {
+    fill: var(--cow-color-text2);
```

Comment:
Here the enum would be used:


```suggestion
    fill: var(${ColorTypes.COLOR_TEXT_2});
```